README.md

```diff
@@ -1,3 +1,61 @@
 # Test262-Stream
 
-Coming soon!
+A [Node.js](https://nodejs.org) API for traversing [the Test262 test
+suite](https://github.com/tc39/test262).
+
+## Usage
+
+```js
+var TestStream = require('test262-stream');
+var stream = new TestStream('/path/to/test262', {
+    // Directory from which to load "includes" files (defaults to the
+    // appropriate subdirectory of the provided `test262Dir`
+    // Optional. Defaults to './harness'
+    includesDir: '/path/to/includes/dir',
+
+    // File system paths refining the set of tests that should be produced;
+    // only tests whose source file matches one of these values (in the case of
+    // file paths) or is contained by one of these paths (in the case of
+    // directory paths) will be created; all paths are interpreted relative to
+    // the root of the provided `test262Dir`
+    // Optional. Defaults to ['test']
+    paths: ['test/built-ins/eval', 'test/language/statements/empty/S12.3_A1.js']
```

Comment:
for the future, probably after this lands, it would be amazing to have an excludePaths setting as well.

Reply:
actually no need for this, I'm now registering tests I want to skip.